logger/logger.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import functools
+import os
+from flask import request
+
+def get_logger(name, log_file, level=logging.INFO):
```

Comment:
make this a class "Logger"

we don't want "stray" functions in our files unless very necessary